packages/framework/src/Component/String/TransformString.php

```diff
@@ -33,6 +33,19 @@ public static function emptyToNull($value)
         return $value === '' ? null : $value;
     }
 
+    /**
+     * @param string|null $value
+     * @return string|null
+     */
+    public static function sanitizeInput(?string $value): ?string
```

Comment:
From the name it is not clear what is being sanitized - that it is empty or whitespace string